Bottom: 22cdce30b3f34eaf3fb19fa68e5f8b17662531a1
Top:    32028a3289cc40130c9fa0b2ebfc5f447dd72523
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-10 10:47:45 +0800

Refresh of tuntap-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index d40583b..a2a2c1e 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1347,11 +1347,8 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 	rxhash = skb_get_hash(skb);
 	skb_queue_tail(&tfile->socket.sk->sk_write_queue, skb);
 
-	if (!more) {
-		local_bh_disable();
+	if (!more)
 		napi_schedule(&tfile->napi);
-		local_bh_enable();
-	}
 
 	stats = get_cpu_ptr(tun->pcpu_stats);
 	u64_stats_update_begin(&stats->syncp);
